Tests/test_SeqIO_QualityIO.py

```diff
@@ -181,7 +181,7 @@ def check_qual_char(self, filename, good_count, count):
          ("long_qual", 3),
          ("short_qual", 2),
          ("double_seq", 3),
-         ("double_qual", 2),
+         ("double_qual", 3),
```

Comment:
This is a functional change  - I think the example contains two good records, then the third record is malformed.

Yes if you only look at the first four lines the third record is OK - but that's exactly the point of the test case, it has a repeated quality block making it malformed.